test-two/src/index.js

```diff
@@ -4,23 +4,29 @@ import { getUsersForOrganisation } from "./services/github";
 const baseUrl = "https://api.github.com/";
 const organisationId = "google";
 const gitHubApiAuthToken = ""; // add your GitHub API OAuth key here to increase request limit
+var pageNumber;
 
 export const go = async () => {
-  try {
-    const users = await getUsersForOrganisation(
-      baseUrl,
-      gitHubApiAuthToken,
-      organisationId
-    );
 
-    const output = users.reduce((result, user) => {
-      return `${result}Username: ${user.login}\n`;
-    }, "");
+    for (pageNumber = 1; pageNumber <= 23; pageNumber++) {
```

Comment:
What will happen when another 100 users work for google?

Reply:
Or if we wanted to change the organisation?

Reply:
Yeah, I was running out of time towards the end because my place had issues with the internet and I was pretty fatigue from a side effect of a vaccine I had. To improve the code it would of been better for me to of stored the list of names in an array and had the loop look for if it started pulling null response.